### PR TITLE
Sort PRs by most recently updated first [v1 - one liner]

### PR DESCRIPTION
**Pull request by calvarez-geny**

### CODE REVIEW

File: gm_pr/prfetcher.py [hunk 1]
```diff
@@ -282,6 +282,8 @@ def get_prs(self):
 
         prlist = [ parse_githubdata(github_data[prid], self.__current_user)
                    for prid in github_data ]
+
+        prlist = sorted(prlist, key = lambda x: x.updated_at, reverse = True)
```
Comment (calvarez-geny) [inline gm_pr/prfetcher.py]: See if you can move this logic to the rendering/view files.  It's a display matter.

Comment (calvarez-geny) [inline gm_pr/prfetcher.py]: Ok.

Comment (calvarez-geny) [inline gm_pr/prfetcher.py]: Here you go: https://github.com/Genymobile/gm_pr/pull/80

**Comment by CedricCabessa** [comment]
https://github.com/Genymobile/gm_pr/pull/80 is merged instead

**Pull request closed without merging**